### DISCUSSION TITLE
Alien Turrets do not fire inside hulls (without motion sensors)

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15011] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

alien turrets don't fire at people if the player is inside a hull. this means interior placed alien turrets do not fire at people in front of them (except weirdly on occasion when directly on top of it?).

this is fixed in ruins by using motion sensors for a limited range of motion, but when outside a hull, they fire at the player normally and accurately even without sensors.

in order to get a full range of motion that fires ONLY when the player is in its crosshairs, you'd need a lot of sensors. You can allow the auto-turret AI to track, and simply use a single large motion sensor for trigger_in, but this will result in it firing even when not yet pointing at the player.

### Reproduction steps

1. place alien turret inside a submarine
2. try to get it to fire at the player (it won't)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.
